themes/edx.org/lms/templates/header.html

```diff
@@ -69,10 +69,9 @@ <h2 class="course-header">
 
     % if user.is_authenticated():
       % if not course or disable_courseware_header:
-        % if not nav_hidden or show_program_listing:
           <nav aria-label="Main" class="nav-main">
             <ul class="left list-inline nav-global authenticated">
-              % if not nav_hidden:
+              % if not nav_hidden and not show_program_listing:
```

Comment:
I have updated condition statements here because when I add `show_program_listing` in context, both previous header and new programs listing were visible.
I think it make sense `% if not nav_hidden and not show_program_listing:
` condition will show old style header, otherwise we will show programs listings tabs.